BindingContext/BPoP.md

```diff
@@ -210,6 +227,8 @@ BPoP: enabled, SameSite=None
 
 The latest configuration replaces the previous configuration. Replacement occurs for the entire config, not just for configuration elements who appear in the `BPoP` header.
 
+Please note that this behavior applies only when there is no enterprise policy override. In case if enterprise policy enabled by the browser config, the BPoP behaves as configured by the policy. More details on key management and storage for enterprise use cases are covered later in this document.
```

Comment:
For my learning, do you know what specific enterprise policy mechanism we'd want to use? Whatever is built into the browser? I was reading about Chrome's policies here, and didn't see a reference to mobile, I think mobile isn't supported: https://support.google.com/chrome/a/answer/9037717?hl=en

Reply:
I can find out, I am not very familiar with it either. Will check with the google engineer and get back.

Reply:
https://chromeenterprise.google/policies/#PasswordManagerEnabled - This has an example for mobile. A policy set should reflect at `chrome://policy/` as per google. Hope this helps.